### ISSUE TITLE
Tailwind is not purging unused styles because no template paths have been provided.

### ISSUE THREAD

**Issue by dwsmart** [nuxt-community/tailwindcss-module#111]
Hi,

When building my nuxt 2.12.2 project, I am getting this warning:

Tailwind is not purging unused styles because no template paths have been provided.

But the css does seem to be purged, I think? Any clues?

**Comment by web2033** [comment]
It puts it in a 1.6Mb js file.
Just did "nuxt generate" on a fresh install.

**Comment by web2033** [comment]
Added purge option to `tailwind.config.js` manually, seems working
```js
module.exports = {
  purge: ["./src/**/*.html", "./src/**/*.vue"],
  theme: {},
  variants: {},
  plugins: []
};
```
Should it be there by default?

**Comment by Triloworld** [comment]
Yep, It isn't by default. 
Proper in my case:

```
  purge: [
    "./pages/**/*.vue", 
    "./components/**/*.vue", 
    "./plugins/**/*.vue",
    "./static/**/*.vue",
    "./store/**/*.vue"
  ],
```

**Comment by santicros** [comment]
It' happened to me too, worked with the @Triloworld solution. But I think it should be added by default!!

**Comment by dwsmart** [comment]
@Triloworld's solution seems to work well! Thanks!

**Comment by santicros** [comment]
I've now seen that @Atinux put the recommended settings on the new [tailwind.config.js](https://github.com/nuxt-community/tailwindcss-module/blob/master/lib/templates/tailwind.config.js)
The problem is it didn't update for current projects.
The recommended settings he put are:
```js
// tailwind.config.js
module.exports = {
  /* ... your actual config */,
  purge: {
    // Learn more on https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
    enabled: process.env.NODE_ENV === 'production',
    content: [
      'components/**/*.vue',
      'layouts/**/*.vue',
      'pages/**/*.vue',
      'plugins/**/*.js',
      'nuxt.config.js'
    ]
  }
}
```

**Comment by dbarjs** [comment]
Thanks @santicros, your config help me a lot. When I build or generate with the @Triloworld solution, the breakpoints like `md:*` don't appears on compiled files.

**Comment by mishingo** [comment]
@dbarjs running into the same issue on my end with `md:*` . Disabled purge on tailwind.config.js and nuxt.config.js

**Comment by callumflack** [comment]
I don't think this module should require having to add in a production flag for `enabled`.

According to the [Tailwind docs](https://tailwindcss.com/docs/controlling-file-size/#enabling-manually), purge should only run in production and they are explicit that `enabled` is a manual override. So this module should not default to requiring a manual override.

![image](https://user-images.githubusercontent.com/749885/84727589-b6051200-afd2-11ea-8546-5e94bd087f51.png)
